### PR TITLE
Required changes to meet pointer type Replicas.

### PR DESCRIPTION
**Pull request by the-redback**

### CODE REVIEW

File: pkg/controller/deployment.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func (c *Controller) ensureDeployment(memcached *api.Memcached) (kutil.VerbType,
 		in.Labels = core_util.UpsertMap(in.Labels, memcached.DeploymentLabels())
 		in.Annotations = core_util.UpsertMap(in.Annotations, memcached.DeploymentAnnotations())
 
-		in.Spec.Replicas = types.Int32P(memcached.Spec.Replicas)
+		in.Spec.Replicas = types.Int32P(*memcached.Spec.Replicas)
```
Comment (tamalsaha) [inline pkg/controller/deployment.go]: Need to check for `nil`. For replicas field, we have not updated the YAML. https://github.com/kubedb/postgres/commit/df3ab70390e1002cb858e3c08c028f917b311bde#diff-50427d6a71cd9c7a69d9eb194186be29R36

Let's when you are back.

**Comment by aerokite** [comment]
This one if fixed with #22

**Comment by tamalsaha** [comment]
@the-redback ,  I think you need to update for replicas > 1 case

**Comment by the-redback** [comment]
Backdated PR. 
Fixed in https://github.com/kubedb/memcached/pull/22.
And initialization will be fixed in Mutation.

**Pull request closed without merging**